vobject/base.py

```diff
@@ -296,7 +299,12 @@ def updateTable(x):
             qp = True
             self.singletonparams.remove('QUOTED-PRINTABLE')
         if qp:
-            self.value = self.value.decode('quoted-printable')
+            if six.PY3:
+                charset = self.params.get('CHARSET', ['UTF-8'])[0]
+                self.value = quopri.decodestring(self.value).decode(charset)
```

Comment:
Let's pull this out for now then. I'd rather have a better idea of why it's in there, and just looking at the code, I'm not seeing why it's needed.